docs/Tailwind/WorkingWithTailwind.stories.mdx

```diff
@@ -0,0 +1,80 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { CATEGORIES } from "../../storybook/constants"
+import { Button } from "@kaizen/button"
+import { Heading } from "@kaizen/typography"
+
+<Meta title={`${CATEGORIES.tailwind}/Working with Tailwind`} />
+
+# Working with Tailwind
+
+This page describes the main intended use cases for this package, and provides examples.
+
+- [Spacing and layouts](#spacing-and-layouts)
+
+<br/>
+
+## Spacing and layouts
+
+For information about Cultureamp's spacing and layout system in general, see [here](#) <LINK_NEEDED>
```

Comment:
Yeah I guess we don't have a great place to link them to - we could always direct them to https://cultureamp.design/guidelines/layout/ but it is woefully outdated 😞

Reply:
This would be `/?path=/story/systems-layout-and-spacing--page`

Reply:
👍